fix: segmentation violation

```
panic: runtime error: invalid memory address or nil pointer dereference                                                                                                                       
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x688f9c]                                                                                                                        
                                                                                                                                                                                              
goroutine 7 [running]:                                                                                                                                                                        
github.com/GEPROG/lassie-bot-dog/plugins/auto_merge.AutoMergePlugin.getStatusComment({0xc000011a58?, 0xc00007b290?, 0xc000020500?, 0xc000116480?}, 0xc0001e1b80, 0x0)                         
    github.com/GEPROG/lassie-bot-dog/plugins/auto_merge/merge_status.go:65 +0x11c                                                                                                             
github.com/GEPROG/lassie-bot-dog/plugins/auto_merge.AutoMergePlugin.updateStatusComment({0xc000011a58?, 0xc00007b290?, 0xc000020500?, 0xc000116480?}, 0x2?, 0x0?, 0x0?)                       
    github.com/GEPROG/lassie-bot-dog/plugins/auto_merge/merge_status.go:49 +0x71                                                                                                              
github.com/GEPROG/lassie-bot-dog/plugins/auto_merge.(*AutoMergePlugin).autoMerge(0xc0000623a0, 0xc0001e1b80, 0xc00037c280)                                                                    
    github.com/GEPROG/lassie-bot-dog/plugins/auto_merge/auto_merge.go:87 +0x3f0                                                                                                               
github.com/GEPROG/lassie-bot-dog/plugins/auto_merge.(*AutoMergePlugin).Execute(0xc0000623a0, 0xc0001e1b80, {0x736cca?})                                                                       
    github.com/GEPROG/lassie-bot-dog/plugins/auto_merge/auto_merge.go:56 +0x46b                                                                                                               
main.loop(0xc000116480)                                                                                                                                                                       
    github.com/GEPROG/lassie-bot-dog/main.go:44 +0x3f9                                                                                                                                        
main.main.func1()                                                                                                                                                                             
    github.com/GEPROG/lassie-bot-dog/main.go:97 +0x1d                                                                                                                                         
reflect.Value.call({0x6be3c0?, 0xc000015230?, 0x0?}, {0x72ad4b, 0x4}, {0x993500, 0x0, 0x6d1501?})                                                                                             
    reflect/value.go:586 +0xb0b                                                                                                                                                               
reflect.Value.Call({0x6be3c0?, 0xc000015230?, 0xc000076b40?}, {0x993500?, 0xc00014e1e0?, 0xf0?})                                                                                              
    reflect/value.go:370 +0xbc                                                                                                                                                                
github.com/go-co-op/gocron.callJobFuncWithParams({0x6be3c0?, 0xc000015230?}, {0x0, 0x0, 0x1?})                                                                                                
    github.com/go-co-op/gocron@v1.28.0/gocron.go:106 +0x237                                                                                                                                   
github.com/go-co-op/gocron.runJob({{{0x0, 0x0}, {0x0, 0x0}}, {0x6be3c0, 0xc000015230}, {0x0, 0x0, 0x0}, 0x0, ...})                                                                            
    github.com/go-co-op/gocron@v1.28.0/executor.go:75 +0x66                                                                                                                                   
github.com/go-co-op/gocron.(*executor).runJob(_, {{{0x0, 0x0}, {0x0, 0x0}}, {0x6be3c0, 0xc000015230}, {0x0, 0x0, 0x0}, ...})                                                                  
    github.com/go-co-op/gocron@v1.28.0/executor.go:162 +0x1d8                                                                                                                                 
github.com/go-co-op/gocron.(*executor).limitModeRunner(0xc00014c120)                                                                                                                          
    github.com/go-co-op/gocron@v1.28.0/executor.go:109 +0x138                                                                                                                                 
created by github.com/go-co-op/gocron.(*executor).run                                                                                                                                         
    github.com/go-co-op/gocron@v1.28.0/executor.go:187 +0x20a                                                                                                                                 
Stream closed EOF for lassie-bot-dog/lassie-bot-dog-6d9558676c-cf5pb (lassie-bot-dog)
```

--- a/plugins/auto_merge/auto_merge.go
+++ b/plugins/auto_merge/auto_merge.go
@@ -82,11 +82,10 @@ func (plugin *AutoMergePlugin) autoMerge(project *gitlab.Project, mergeRequest *
 		status.err = err.Error()
 		plugin.updateStatusComment(project, mergeRequest, status)
 		return
-	} else {
-		log.Info("merged >>>", squashMessage)
-		status.merged = true
 	}
 
+	log.Info("merged >>>", squashMessage)
+	status.merged = true
 	plugin.updateStatusComment(project, mergedMergeRequest, status)
 }
 

--- a/plugins/auto_merge/auto_merge.go
+++ b/plugins/auto_merge/auto_merge.go
@@ -78,7 +78,10 @@ func (plugin *AutoMergePlugin) autoMerge(project *gitlab.Project, mergeRequest *
 
 	mergedMergeRequest, _, err := plugin.Client.MergeRequests.AcceptMergeRequest(project.ID, mergeRequest.IID, acceptMergeRequestOptions)
 	if err != nil {
-		log.Debug("Can't merge", err)
+		log.Error("Can't merge", err)
+		status.err = err.Error()
+		plugin.updateStatusComment(project, mergeRequest, status)
+		return
 	} else {
 		log.Info("merged >>>", squashMessage)
 		status.merged = true

--- a/plugins/auto_merge/auto_merge.go
+++ b/plugins/auto_merge/auto_merge.go
@@ -79,7 +79,7 @@ func (plugin *AutoMergePlugin) autoMerge(project *gitlab.Project, mergeRequest *
 	mergedMergeRequest, _, err := plugin.Client.MergeRequests.AcceptMergeRequest(project.ID, mergeRequest.IID, acceptMergeRequestOptions)
 	if err != nil {
 		log.Error("Can't merge", err)
-		status.err = err.Error()
+		status.err = err
 		plugin.updateStatusComment(project, mergeRequest, status)
 		return
 	}

--- a/plugins/auto_merge/merge_checks.go
+++ b/plugins/auto_merge/merge_checks.go
@@ -25,6 +25,7 @@ type mergeStatus struct {
 	checkResults    []*mergeCheckResult
 	merged          bool
 	allChecksPassed bool
+	err             string
 }
 
 func (plugin AutoMergePlugin) checkMergeRequest(project *gitlab.Project, mergeRequest *gitlab.MergeRequest) *mergeStatus {

--- a/plugins/auto_merge/merge_checks.go
+++ b/plugins/auto_merge/merge_checks.go
@@ -25,7 +25,7 @@ type mergeStatus struct {
 	checkResults    []*mergeCheckResult
 	merged          bool
 	allChecksPassed bool
-	err             string
+	err             error
 }
 
 func (plugin AutoMergePlugin) checkMergeRequest(project *gitlab.Project, mergeRequest *gitlab.MergeRequest) *mergeStatus {

--- a/plugins/auto_merge/merge_status.go
+++ b/plugins/auto_merge/merge_status.go
@@ -28,10 +28,10 @@ func encodeMergeCheckStatus(mergeCheck mergeCheck, status *mergeStatus) string {
 func (plugin AutoMergePlugin) encodeMergeStatus(status *mergeStatus) string {
 	comment := authorTag + "\n"
 
-	if status.err != "" {
+	if status.err != nil {
 		comment = comment + ":warning: I am sorry but I sniffed something strange while checking your merge request:\n"
 		comment = comment + "```\n"
-		comment = comment + status.err + "\n"
+		comment = comment + status.err.Error() + "\n"
 		comment = comment + "```\n"
 		comment = comment + "---\n"
 	}

--- a/plugins/auto_merge/merge_status.go
+++ b/plugins/auto_merge/merge_status.go
@@ -29,7 +29,7 @@ func (plugin AutoMergePlugin) encodeMergeStatus(status *mergeStatus) string {
 	comment := authorTag + "\n"
 
 	if status.err != "" {
-		comment = comment + ":dog: I am sorry but I sniffed an error while checking your merge request:\n"
+		comment = comment + ":warning: I am sorry but I sniffed something strange while checking your merge request:\n"
 		comment = comment + "```\n"
 		comment = comment + status.err + "\n"
 		comment = comment + "```\n"

--- a/plugins/auto_merge/merge_status.go
+++ b/plugins/auto_merge/merge_status.go
@@ -28,6 +28,14 @@ func encodeMergeCheckStatus(mergeCheck mergeCheck, status *mergeStatus) string {
 func (plugin AutoMergePlugin) encodeMergeStatus(status *mergeStatus) string {
 	comment := authorTag + "\n"
 
+	if status.err != "" {
+		comment = comment + ":dog: I am sorry but I sniffed an error while checking your merge request:\n"
+		comment = comment + "```\n"
+		comment = comment + status.err + "\n"
+		comment = comment + "```\n"
+		comment = comment + "---\n"
+	}
+
 	if status.merged {
 		comment = comment + ":dog: Thank you for your contribution. Always nice to have some helping hands :feet:\n"
 		comment = comment + "---\n"

